pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/OutputColumnAstNode.java

```diff
@@ -48,4 +57,35 @@ public void updateBrokerRequest(BrokerRequest brokerRequest) {
       }
     }
   }
+
+  @Override
+  public void updatePinotQuery(PinotQuery pinotQuery) {
+    for (AstNode astNode : getChildren()) {
+      // If the column is a function call, it must be an aggregation function
+      if (astNode instanceof FunctionCallAstNode) {
+        FunctionCallAstNode node = (FunctionCallAstNode) astNode;
+        Expression functionExpr;
+        if (node.getName().equalsIgnoreCase("count")) {
+          // COUNT aggregation function always works on '*'
+          Expression expression;
+          expression = new Expression(ExpressionType.IDENTIFIER);
+          expression.setIdentifier(new Identifier("*"));
```

Comment:
should this be an identifier or literal?

Reply:
identifier imo